Makefile

```diff
@@ -0,0 +1,34 @@
+# Get protoc commands
+PROTOC := $(shell command -v protoc)
+PROTOC_GEN_GO := $(shell command -v protoc-gen-go)
+PROTOC_GEN_GO_GRPC := $(shell command -v protoc-gen-go-grpc)
+
```

Comment:
If found, you need to check for the version number and if it meets the minimum requirements.